editoast/src/models/infra.rs

```diff
@@ -289,6 +289,45 @@ impl List<NoParams> for Infra {
     }
 }
 
+#[async_trait]
+impl Delete for Infra {
+    async fn delete_conn(conn: &mut PgConnection, infra_id: i64) -> Result<bool> {
+        use crate::tables::infra_object_track_section::dsl as track_section_dsl;
+        use crate::tables::search_track::dsl as search_track_dsl;
+
+        sql_query("ALTER TABLE infra_object_track_section DISABLE TRIGGER search_track__del_trig")
+            .execute(conn)
+            .await
+            .expect("Failed to disable trigger");
```

Comment:
Since the function returns a `Result`, why would you want to panic here in case the operation fail? Maybe just put `?` instead?

Same remark for all the `.expect()` below.

Reply:
Yes, you are right 👍🏽